ops/services/app_functions/report_stream_batched_publisher/functions/config.ts

```diff
@@ -22,6 +22,7 @@ export const ENV = (() => {
     TEST_EVENT_QUEUE_NAME: "storage queue resource name for Test Events",
     REPORTING_EXCEPTION_QUEUE_NAME:
       "storage queue resource name for exceptions in publishing to ReportStream",
+    PUBLISHING_ERROR_QUEUE_NAME: "storage queue resource name for HTTP 400 errors in publishing to ReportStream",
```

Comment:
teeeechnically it's not a queue for the errors it's a queue for the test events that cause 400 errors

Reply:
you are teeeechnically correct - the best kind of correct